.github/actions/windows-run_examples/action.yml

```diff
@@ -19,8 +19,16 @@ runs:
         $errchk = (Get-ChildItem -Path ./| Where-Object { $_.Name -like "*_example.exe"} | Measure-Object).Count
         if( $errchk -gt 0){
           Write-Output "Running windows Examples."
-          Get-ChildItem -Path ./| Where-Object { $_.Name -like "*_example.exe"} | Foreach { Write "Running $_"; if( $(Start-Process $_.FullName -Passthru -Wait).ExitCode -ne 0){Write "Programm terminated with error, abort."; Exit 1}}
-        } else {
-          Write-Output "No Examples found, abort."	
+          Get-ChildItem -Path ./| Where-Object { $_.Name -like "*_example.exe" } | ForEach-Object {
+            Write-Output "Running $($_.Name)"
+            & $_.FullName
```

Comment:
Is there maybe an alias for `&`, or a similar command to run the example? So that this looks less like an incantation

Reply:
unfortunately not :)